src/implicit_restart.jl

```diff
@@ -88,6 +88,10 @@ function single_shift!(H_whole::AbstractMatrix{Tv}, min, max, μ::Tv, Q::Abstrac
     @inbounds H[n, n - 1] = H[n + 1, n - 1]
     @inbounds H[n + 1, n - 1] = zero(Tv)
 
+    # Update Q with the last rotation
```

Comment:
I don't think this belongs here. We're not using this column?

Reply:
Right. Maybe the tests for which these lines are necessary should be removed

Reply:
You can move it to the tests ;)